README.md

```diff
@@ -9,6 +9,37 @@ This repository contains the Strimzi canary tool implementation.
 It acts as an indicator of whether Kafka clusters are operating correctly.
 This is achieved by creating a canary topic and periodically producing and consuming events on the topic and getting metrics out of these exchanges.
 
+## Deployment
+
+In order to deploy the Strimzi canary in the Kubernetes cluster alongside the already running Apache Kafka cluster, first download the installation files from one of the [releases](https://github.com/strimzi/strimzi-canary/releases) you are interested in.
+Unzip the release package and edit the `Deployment` in order to specify the right bootstrap servers for connecting to the Apache Kafka cluster; it is defined by the `KAFKA_BOOTSTRAP_SERVERS` environment variable.
+The provided `Deployment` has also TLS disabled by default (`TLS_ENABLED` as `false`) and the canary producing and consuming messages every 10 seconds (`RECONCILE_INTERVAL_MS` as `"10000"`).
+
+Then just run the following command:
+
+```shell
+kubectl apply -f ./install
+```
+
+Other than creating the corresponding `Deployment`, the canary will run with a specific `ServiceAccount` and a `Service` will be created in order to make the Prometheus metrics accessible via HTTP on port `8080`.
+
+### Encryption and TLS authentication
```

Comment:
Should this maybe be split into two sections? One for encryption and one for TLS client auth? It would make it more clear that you can also use encryption with SASL.

Reply:
@scholzj agree. I just changed by having an "Encryption and TLS" section (just for TLS) separated from the "Authentication and authorization" with TLS client auth and SASL auth.